### DISCUSSION TITLE
new TTGO Beamboard T22V1.1 2019.1212 with Lora SX1262

### DISCUSSION THREAD

**Discussion by hansju-11** [cyberman54/ESP32-Paxcounter#791] [Ideas]
should we had support for this board? Question points to the IMB LMIC STACK. Today we have no support for the SX1262 chip set. 

https://github.com/meshtastic/Meshtastic-device has done some integration. Maybe we can benefit from that project.

**Comment by cyberman54** [comment]
Paxcounter is based on LoRaWAN using the MCCI LMIC stack as radio control software. MCCI LMIC has, so far, no support for SX1262 radio, so it's not possible.
Meshtastic is not a LoRaWAN based radio, so does not help here.

**Comment by hansju-11** [comment]
ok, I will check https://github.com/Xinyuan-LilyGO/LilyGO-T-Beam how they will deal with LoraWan and SX1262.

I will come back to that.

 

kind regards 

Hans-Jürgen

 
 

Gesendet: Sonntag, 11. April 2021 um 21:10 Uhr
Von: "Verkehrsrot" ***@***.***>
An: "cyberman54/ESP32-Paxcounter" ***@***.***>
Cc: "hansju-11" ***@***.***>, "Author" ***@***.***>
Betreff: Re: [cyberman54/ESP32-Paxcounter] new TTGO Beamboard T22V1.1 2019.1212 with Lora SX1262 (#791)


 

Paxcounter is based on LoRaWAN using the MCCI LMIC stack as radio control software. MCCI LMIC has, so far, no support for SX1262 radio, so it's not possible.
Meshtastic is not a LoRaWAN based radio, so does not help here.

**Reply by cyberman54** [reply]
There is no LORAWAN stack in TTGO Repo for SX1262, only a Radiolib Example (which is not LORAWAN).

**Comment by Max-Plastix** [comment]
This is good information to get out there.  I went down this path when I mistakenly bought the T-Beam "with IPEX/u.FL connector".  It has a superior uBlox M8 GNSS receiver, IPEX instead of SMA antenna, and indeed the SX1262.   I didn't do enough research and bumped my head into the same surprise missing LMIC library support for it.

It might be worth calling this out on the top-level README for ESP32-Paxcounter?

For now, it rests on the shelf looking forward to LMIC support.  There is some statement that SX1262 has improved battery performance and no deep incompatibility, so perhaps in the future, we can use it.

**Reply by cyberman54** [reply]
you're welcome to create a PR to change the readme.md
